plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/SymbolManager.java

```diff
@@ -78,8 +81,11 @@
   private GeoJsonSource geoJsonSource;
   private SymbolLayer symbolLayer;
 
+  // callback listeners
+  private OnSymbolClickListener symbolClickListener;
```

Comment:
To stay consistent with listeners we expose in the Maps SDK it'd be great to have a list here.